fdk-organization-bff.yaml

```diff
@@ -54,19 +54,6 @@ components:
       x-examples:
         example-fylke:
           organization:
-            organisasjonsform: FYLK
-            naeringskode:
-              uri: 'http://www.ssb.no/nace/sn2007/84.110'
-              code: '84.110'
-              prefLabel:
-                'no': Generell offentlig administrasjon
-            sektorkode:
-              uri: 'http://www.brreg.no/sektorkode/6500'
-              code: '6500'
-              prefLabel:
-                'no': Kommuneforvaltningen
-            valid: true
-            uri: 'http://data.brreg.no/enhetsregisteret/enhet/971045698'
             id: '971045698'
             name: ROGALAND FYLKESKOMMUNE
```

Comment:
Vi treng vel ikkje både `name` og `prefLabel`. Vi burde gitt kun feks `prefLabel`.

Reply:
Virksomhetsnavn viser prefLabel hvis den finnes fra publisher-endepunktet. For mange virksomheter finnes ikke prefLabel, da vises name. Men om du gjør den mappingen i backend så er det greit.